src/stylus/main.styl

```diff
@@ -0,0 +1,113 @@
+@import '~vuetify/src/stylus/settings/_colors'
+$font-size-table := 18px
+
+$material-light := {
```

Comment:
Do you have to commit all of this? Can't you just patch the existing object with the changes needed?